requirements.txt

```diff
@@ -11,7 +11,7 @@ imageio~=2.19
 joblib~=1.0
 matplotlib>=3.3.0
 nibabel~=3.2
-onnxruntime==1.7.0
+onnxruntime~=1.12.0
```

Comment:
You're totally right. I was thinking that `~=1.8.0` wouldn't work because there's no 1.8.x build for python310. But `~=1.8` means something different, and it does work on python310:

```
p115628@joplin:~$ pip install 'onnxruntime~=1.8'
Defaulting to user installation because normal site-packages is not writeable
Collecting onnxruntime~=1.8
  Downloading onnxruntime-1.12.0-cp310-cp310-manylinux_2_27_x86_64.whl (4.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.9/4.9 MB 38.7 MB/s eta 0:00:00
```

So :+1: